src/scripts/librarymenu.js

```diff
@@ -177,6 +186,32 @@ define(["dom", "layoutManager", "inputManager", "connectionManager", "events", "
         });
     }
 
+    function onSyncButtonClicked() {
+        var btn = this;
+
+        require(["groupSelectionMenu"], function (groupSelectionMenu) {
+            groupSelectionMenu.show(btn);
+        });
+    }
+
+    function updateSyncplayIcon(event, enabled) {
+        var icon = headerSyncButton.querySelector("i");
```

Comment:
please no, icons should be `<span>`-s now, and the image should be selected by class, not by `.innerHTML` because innerHTML breaks on certain platforms like WebOS. cc @MrTimscampi @dmitrylyzo